### PR TITLE
Added clarifications for allow for HCP and malware

### PR DESCRIPTION
**Pull request by dhairyya**
1. Diverted customer to admin submission
2. Made things clear about allow in Tenant allow/block list

### CODE REVIEW

**Comment by dhairyya** [comment]
@chrisda  could you please approve this for me?

File: microsoft-365/security/office-365-security/secure-by-default.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ Because Microsoft wants to keep our customers secure by default, some tenants ov
 - IP Allow List (connection filtering)
 - Exchange mail flow rules (also known as transport rules)
 
+If you might want to temporarily allow certain messages that are still being blocked by Microsoft, do it [via Admin submissions](https://learn.microsoft.com/microsoft-365/security/office-365-security/submissions-admin?view=o365-worldwide#report-good-email-to-microsoft).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/secure-by-default.md]:
```suggestion
If you want to temporarily allow certain messages that are still being blocked by Microsoft, do it [via Admin submissions](/microsoft-365/security/office-365-security/submissions-admin#report-good-email-to-microsoft). 
```

File: microsoft-365/security/office-365-security/secure-by-default.md [hunk 2]
```diff
@@ -75,4 +77,4 @@ You should only consider using overrides in the following scenarios:
...
-- False positives: You might want to temporarily allow certain messages that are still being analyzed by Microsoft [via Admin submissions](submissions-admin.md). As with all overrides, it is recommended that they are temporary.
+- False positives: You might want to temporarily allow certain messages that are still being blocked by Microsoft, do it [via Admin submissions](https://learn.microsoft.com/microsoft-365/security/office-365-security/submissions-admin?view=o365-worldwide#report-good-email-to-microsoft). By default, allow entries for domains and email addresses, files and URLs are created for 30 days, while allow entries for spoofed senders never expire. Microsoft will either learn from the allow entries for domains and email addresses, files and URLs within those 30 days, or automatically extend it for you.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/secure-by-default.md]:
```suggestion
- False positives: If you want to temporarily allow certain messages that are still being blocked by Microsoft, do it [via Admin submissions](/microsoft-365/security/office-365-security/submissions-admin#report-good-email-to-microsoft). By default, allow entries for domains and email addresses, files and URLs are created for 30 days, while allow entries for spoofed senders never expire. Microsoft will either learn from the allow entries for domains and email addresses, files, and URLs within those 30 days, or automatically extend it for you.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dhairyya, kindly commit the suggested changes. Many thanks.

**Pull request merged**